app/components/post_list/post_list.js

```diff
@@ -183,6 +183,7 @@ export default class PostList extends PureComponent {
                 theme={theme}
                 getItem={this.getItem}
                 getItemCount={this.getItemCount}
+                contentContainerStyle={{paddingTop: 5}}
```

Comment:
Should this use a StyleSheet?

Reply:
Wasn't really sure on that. I suppose it should, but it's the only style in the entire file so I'm 50/50 on whether it should use it or not.

Reply:
I'm not sure how FlatList works internally, but this might the container to re-render every time PostList does because it's passing a different object for contentContainerStyle each time. I'd prefer to use a StyleSheet so that it doesn't do that